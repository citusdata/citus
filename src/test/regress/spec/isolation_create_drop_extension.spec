setup
{
	SELECT citus_internal.restore_isolation_tester_func();
	DROP EXTENSION citus;
}

teardown
{
	SELECT citus_internal.replace_isolation_tester_func();

	// Restore a node for later tests to succeed.
	SELECT 1 FROM master_add_node('localhost', 57637);
}

session "s1"

step "s1-create"
{
	CREATE EXTENSION citus;
}

session "s2"

step "s2-check"
{
	 SELECT 1 FROM citus_version();
}

// session 1 creates the extension, s2 checks if it exists.
permutation "s1-create" "s2-check"
