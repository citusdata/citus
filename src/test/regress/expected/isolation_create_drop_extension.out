Parsed test spec with 2 sessions

starting permutation: s1-create s2-check
step s1-create:
 CREATE EXTENSION citus;

step s2-check:
  SELECT 1 FROM citus_version();

?column?
---------------------------------------------------------------------
       1
(1 row)

?column?
---------------------------------------------------------------------
       1
(1 row)

